matching-service/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:14-alpine
```

Comment:
is there a reason for some services to use specific node 18, 17, 14?

Reply:
No difference for now.